code/game/objects/items/tcg/tcg_machines.dm

```diff
@@ -95,11 +119,14 @@ GLOBAL_LIST_EMPTY(tcgcard_machine_radial_choices)
 	if(card_template)
 		. += span_notice("There is currently a [card_template.name] card inserted.")
 	else
-		. += span_notice("There is no card currently inserted.")
+		if(current_summon)
+			. += ("There is currently a blank card inserted.")
```

Comment:
```suggestion
			. += "There is currently a blank card inserted."
```

No need for the parenthesis if you aren't going to span macro it. If you did want to span macro it, put that in.